OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -60,15 +60,19 @@ public AttackBase(Actor self, AttackBaseInfo info)
 			this.self = self;
 		}
 
-		void INotifyCreated.Created(Actor self)
+		protected override void Created(Actor self)
```

Comment:
This inherits from `ConditionalTrait` which also implements `INotifyCreated` and already exposes a virtual method to override.  What @hartmark has done here is correct, and the old code was more bogus than we first thought.